cmake/projects/Boost/schemes/url_sha1_boost_library.cmake.in

```diff
@@ -289,6 +289,11 @@ if("@MSVC@" AND is_x64)
   list(APPEND build_opts "address-model=64")
 endif()
 
+string(COMPARE EQUAL "@HUNTER_MSVC_ARCH@" "x86" is_x86)
```

Comment:
What about previous version of Boost? Will this work with 1.64 for example?

Reply:
Please update `PACKAGE_INTERNAL_DEPS_ID` of all Boost components.

Reply:
> What about previous version of Boost? Will this work with 1.64 for example?

It should work fine as far as I can tell, the option exists since forever and only default behavior have changed. I've compiled boost example locally with my hunter master and forced version to 1.64 it seems to work. If you wish I can also re-run all tests with 1.64.

> Please update PACKAGE_INTERNAL_DEPS_ID of all Boost components.

Fixed.